com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs

```diff
@@ -62,26 +116,25 @@ public static void ReconnectStreamsWithPath(string path)
         AlembicStreamDescriptor m_streamDesc;
         AlembicTreeNode m_abcTreeRoot;
         aiConfig m_config;
-        aiContext m_context;
+        ContextPair context;
```

Comment:
Why the name change? m_context to context

Reply:
I hate variables starting with m :) Fixed for consistency.